api/preprint_providers/views.py

```diff
@@ -28,12 +28,21 @@ class PreprintProviderList(JSONAPIBaseView, generics.ListAPIView, ODMFilterMixin
 
     OSF Preprint Providers have the "preprint_providers" `type`.
 
-        name           type               description
-        =========================================================================
-        name           string             name of the preprint provider
-        logo_path      string             a path to the preprint provider's static logo
-        banner_path    string             a path to the preprint provider's banner
-        description    string             description of the preprint provider
+        name                 type                description
+        ==========================================================================================================
+        name                 string              name of the preprint provider
```

Comment:
Since this PR was started, `developer.osf.io` has come into existence. Will this documentation need to be captured in the new source of record?